dhis-2/dhis-api/src/main/java/org/hisp/dhis/user/UserSettingKey.java

```diff
@@ -55,6 +59,19 @@
     private final Serializable defaultValue;
 
     private final Class<?> clazz;
+    
+    private static Map<String, Serializable> DEFAULT_USER_SETTINGS_MAP = new HashMap<>();
+
+    static
+    {
```

Comment:
Instead of using static blocks I think you can write this using streams, something like:

DEFAULT_USER_SETTINGS_MAP = UserSettingKey.values().filter( k -> k.defaultValue() != null ).collect( Collectors.toMap( UserSettingKey::getName, UserSettingKey::getDefaultValue ) );